readme/phpstorm.md

```diff
@@ -0,0 +1,27 @@
+BLT includes several components that can be used with PHPStorm:
+1. An editor config file
+2. A set of PHPCS rules and associated configuration
+3. DrupalVM with Xdebug configuration
+
+Here is the recommended setup process for new projects in PHPStorm.
+
+## Setting up inspections
+
+PHPStorm comes with a whole ton of built-in inspections for PHP. You can enable / disable each one individually in the UI. Most of these make sense in the context of Drupal / BLT projects, but not all.
+
+To ensure consistent validation across all environments on your project, it's better to disable PHPStorm's built-in inspections and instead delegate code sniffing to Coder and PHPCodeSniffer, using the phpcs.xml rulesets that you distribute with your project (and that BLT provides by default).
+
+Specifically:
```

Comment:
Should this include a note about the version of PHPStorm for which these instructions are written? I can see someone on an older version getting confused if the UI is different.

Reply:
I added a note